registry/github/template.go

```diff
@@ -56,5 +56,5 @@ func (c *client) Template(u *library.User, s *registry.Source) ([]byte, error) {
 		return []byte(strData), nil
 	}
 
-	return nil, fmt.Errorf("no valid template found at %s/%s/%s", s.Org, s.Repo, s.Name)
+	return nil, fmt.Errorf("no Vela template found at %s/%s/%s", s.Org, s.Repo, s.Name)
```

Comment:
Can this be updated to include the branch that Vela attempted to pull the file from?